### PR TITLE
feat/PRO-2654/loading-screen

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- Changed the apps loading screen to a text and fade in app page using React Springs

## How Has This Been Tested?

- Existing unit tests and manual testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/pages/App/index.tsx [hunk 1]
```diff
@@ -1,38 +1,80 @@
...
+import { AppManifest } from '../../types';
+
+type AnimatedAppTitleProps = {
+  text: string;
+};
+
+const AnimatedAppTitle: React.FC<AnimatedAppTitleProps> = ({ text }) => {
+  const [isDisplaying, setIsDisplaying] = useState(true);
+
+  const letters = text.split('');
```
Comment (IAmKio) [inline src/pages/App/index.tsx:21]: i noticed that the loading screen text is missing spaces, i know it’s unlikely to be due to this line but could you check this please?

File: src/pages/App/index.tsx [hunk 2]
```diff
@@ -1,38 +1,80 @@
...
+
+  const trail = useTrail(letters.length, {
```
Comment (IAmKio) [inline src/pages/App/index.tsx:23]: can we alter the duration here to 2 seconds, with the build-in and build-out to 250ms? Just to further refine the animation - the app name screen is too long at 4 seconds

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few small animation refinements but looks and feels now so much better!

File: src/pages/App/index.tsx (hunk 1 above)
Comment (RanaBug) [reply src/pages/App/index.tsx:21]: Yes I know, I checked and the space is included in this split... I am investigating further

File: src/pages/App/index.tsx [hunk 3]
```diff
@@ -1,38 +1,85 @@
...
+    from: { opacity: 0, transform: 'translateY(32px)' },
+    to: {
+      opacity: isDisplaying ? 1 : 0,
+      transform: isDisplaying ? 'translateY(0px)' : 'translateY(32px)',
+    },
+    config: {
+      tension: 210,
+      friction: 20,
+      mass: 1,
+      duration: 25, // equivalent to 250 ms
```
Comment (IAmKio) [inline src/pages/App/index.tsx:33]: Well done! 🚀

**Review by IAmKio — APPROVED** [review]
LGTM - great work with React Spring! 🚀

**Pull request merged**